src/Silex/Application.php

```diff
@@ -89,7 +89,7 @@ public function __construct(array $values = array())
 
         $this['dispatcher_class'] = 'Symfony\\Component\\EventDispatcher\\EventDispatcher';
         $this['dispatcher'] = $this->share(function () use ($app) {
-            /**
+            /*
```

Comment:
this should be kept as a docComment, as it is really one. the real fix would be to make it valid: ``@var EventDispatcherInterface $dispatcher``